src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -201,216 +188,44 @@ function epidemicModelReducer(
-      if (facilityModel || state.countyLevelDataLoading) {
-        return;
-      }
-
-      const persistedState = pick(state, persistedKeys);
-      replaceHistoryState(persistedState);
-
-      if (state.stateInitialized) saveState(persistedState);
-    },
-    // replaceHistoryState changes on every render so must be excluded
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [state],
-  );
-
-  // fetch from external datasource
-  React.useEffect(
-    () => {
-      async function effect() {
-        try {
-          const response = await fetch(
-            "https://docs.google.com/spreadsheets/d/e/2PACX-1vSeEO7JySaN21_Cxa7ON_x" +
-              "UHDM-EEOFSMIjOAoLf6YOXBurMRXZYPFi7x_aOe-0awqDcL4KZTK1NhVI/pub?gid=" +
-              "1836987932&single=true&output=csv",
-          );
-          let rawCSV = await response.text();
-          // the first line is not the header row so we need to strip it
-          rawCSV = rawCSV.substring(rawCSV.indexOf("\n") + 1);
-
-          const parsedArray = csvParse(
-            rawCSV,
-            (row): CountyLevelRecord => {
-              // we only need a few columns, which we will explicitly format
-
-              // defaulting missing numbers to zero will not produce meaningful modeling
-              // but it should at least prevent the UI from breaking;
-              // users can still substitute their own values via form inputs
-              const reportedCases: number = numeral(row.cases).value() || 0;
-              const estimatedTotalCases: number =
-                reportedCases * (1 / caseReportingRate);
-              // TODO: distinguish jail vs prison?
-              const totalIncarceratedPopulation: number =
-                numeral(row["Total Incarcerated Population"]).value() || 0;
-              const totalPopulation: number =
-                numeral(row["Total Population"]).value() || 0;
-
-              return {
-                county: row.County || "",
-                state: row.State || "",
-                hospitalBeds: numeral(row["Hospital Beds"]).value() || 0,
-                totalIncarceratedPopulation,
-                estimatedIncarceratedCases: Math.round(
-                  totalPopulation
-                    ? (totalIncarceratedPopulation / totalPopulation) *
-                        estimatedTotalCases
-                    : 0,
-                ),
-              };
-            },
-          )
-            // rows without County are known to be junk
-            .filter((row) => row.county !== "");
-
-          const nestedStateCounty = rollup(
-            parsedArray,
-            // there will only ever be one row object per county
-            (v: object[]) => v[0],
-            (d: DSVRowAny) => d.state as string,
-            (d: DSVRowAny) => d.county as string,
-            // some wrong/outdated typedefs for d3 are making typescript sad
-            // but this should check out
-          ) as CountyLevelData;
-
```

Comment:
The first `useEffect` call and lines 347 - 413 were all deeply confusing to me so I'm really glad to see they're gone 🥳